RemoteWellnessSupportApp/Activity/PhysicalCondition/ViewModels/PhysicalConditionEntryFormViewModel.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  PhysicalConditionEntryFormViewModel.swift
+//  RemoteWellnessSupportApp
+//
+//  Created by 岩本雄貴 on 2024/02/17.
+//
+
+import Combine
+import Foundation
+import SwiftData
+
+class PhysicalConditionEntryFormViewModel: ObservableObject {
+    @Published var selectedDateTime = Date()
+    @Published var memo = ""
+    @Published var selectedRating: PhysicalConditionRating?
+    @Published var isErrorAlert: Bool = false
+    @Published var errorMessage: String = ""
+    let successPublisher = PassthroughSubject<Void, Never>()
+
+    func insertPhysicalCondition(_ modelContext: ModelContext) {
+        guard validateInputs() else {
+            return
+        }
+
+        let rating = selectedRating!.rawValue
```

Comment:
強制アンラップはクラッシュの原因になるので極力避けることが望ましいです。

```suggestion
        guard let rating = selectedRating?.rawValue else { return }
```

Reply:
下記にて修正しました
https://github.com/y-iwamoto/RemoteWellnessSupportApp/pull/6/commits/2a78f487831c609b7488810ef07669afb9e9c7e4